src/docs/getting-started/usage.mdx

```diff
@@ -0,0 +1,136 @@
+---
+name: Usage
+menu: 'Getting Started'
+route: /getting-started/usage
+---
+
+# Usage
+
+## Responsive Meta Tag
+
+React UI is designed and developed mobile-first, a strategy in which we first
+write code for mobile devices, and then scale up components as necessary using
+CSS media queries. To ensure proper rendering and touch zooming for all devices,
+add the responsive viewport meta tag to your `<head>` element:
+
+```html
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+```
+
+## Fonts
+
+React UI is designed with the Titillium Web font. Add it to your project eg. via
+Google Fonts in your `<head>` element:
+
+```html
+<link
+  href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300;400;600;700&display=swap"
+  rel="stylesheet"
+/>
+```
+
+Remember to include all necessary font weights (and only them — for better
+performance).
+
+## Foundation CSS
+
+React UI honors ITCSS principles to make sure that its CSS code base will work
+and perform well even in large scale. There are two global CSS layers:
+
+- **Theme:** a collection of hundreds of CSS custom properties that define the
+  look of your app. See [Theming](/getting-started/theming) for more.
+- **Foundation:** the ground-zero CSS for React UI components. Includes global
+  resets and fixes rendering inconsistencies across browsers with
+  `normalize.css`. Not to be confused with the Foundation CSS framework.
+
+Import the two global CSS layers as early as possible in your document:
+
+```js
+import '@react-ui-org/react-ui/src/lib/theme.scss';
+import '@react-ui-org/react-ui/src/lib/foundation.scss';
+```
+
+⚠️ As of now, you will need **working SASS pipeline** for this to work. This is
+to be changed to a ready-to-use CSS in the future.
+
+️👉️ Because of using CSS custom properties, it is technically possible to import
```

Comment:
> The less we have the better.

Actually, the opposite is true. I can't think of any documentation which I'd dislike for its excessive length and too many details (maybe I prefer MDN over W3C specs to be fair), but I very often struggle with documentations that are too brief. Last time was today and all yesterday to give example (during work on release automation with third-party GitHub actions, see #71).

Reply:
And as for this line in the docs: I'd leave it here. It's a question that I can perfectly imagine rising on frontend developers' minds: should I include the custom properties before, or after the foundation styles? I have been facing it since I started using LESS and then SASS — overriding their variables works differently which can confuse me even today.